### ISSUE TITLE
Suggestion: Make silverwood leaves drop quicksilver drops

### ISSUE THREAD

**Issue by Hyxaru** [Azanor/thaumcraft-suggestions#1]
I can imagine the quicksilver from the leaves being some kind of leave sap. Therefor smaller, more common quicksilver drops might be more appealing then big drops of quicksilver.

**Comment by Fortanono** [comment]
This is a really cool idea right here.

**Comment by darthvader45** [comment]
Yeah, similar to how oak and dark oak leaves have a rare chance to drop apples.  Maybe a similar drop chance for this.

**Comment by Azanor** [comment]
Very good point.

**Comment by MalkContent** [comment]
I like this. You end up with a looooot of quicksilver once you start even a small (4 tree) Silverwood node farm. If it's just droplets it'd not be so over the top. Especially considering that you can easily turn that stuff into other metals with transmutation.
